projects/packages/forms/src/contact-form/js/accessible-form.js

```diff
@@ -354,9 +373,17 @@ const createSpinner = () => {
  */
 const createWarningIcon = () => {
 	const elt = document.createElement( 'span' );
+	const srOnly = document.createElement( 'span' );
```

Comment:
I've update the function implementation to make sure the exclamation mark is not read by screen readers.